tests/acceptance/customCommands/filesListScrollToTop.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Scroll the files list to the beginning
+ */
+exports.command = async function () {
```

Comment:
feels like that should go into the filesList pageObject

Reply:
Ok, will move